catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -366,4 +378,19 @@
         </plugins>
 
     </build>
+
+    <profiles>
+        <profile>
+            <id>webpack dev</id>
+            <activation>
+                <property>
+                    <name>skipStatic</name>
```

Comment:
Talked to @djblue , agree that it is less than ideal. I think there is a lot we could do to make our builds less confusing. Ideally we would just have one `-Ddevmode` flag instead of the list we have now. I did not want to make that list any longer.